modules/@apostrophecms/log/index.js

```diff
@@ -134,28 +145,28 @@ module.exports = {
           return;
         }
         // Add environment specific severity levels if no severity is specified.
-        if (!self.filters['*'].severity || self.filters['*'].severity.length === 0) {
+        if (!self.filters['*'].severity) {
           self.filters['*'] = {
             ...self.filters['*'],
             severity: self.getDefaultSeverity(process.env.NODE_ENV === 'production')
           };
         }
 
         // Handle wildcards and validate.
-        Object.keys(self.filters).forEach((module) => {
-          Object.keys(self.filters[module]).forEach((type) => {
-            if (self.filters[module][type] === true || self.filters[module][type] === '*') {
-              self.filters[module][type] = type === 'severity'
+        for (const [ module, config ] of Object.entries(self.filters)) {
+          for (const [ type, value ] of Object.entries(config)) {
+            if (value === true || value === '*') {
+              self.filters[module][type] = (type === 'severity')
```

Comment:
You can assign to `config[type]` here, no?

Reply:
Fixed.